stats.sql

```diff
@@ -0,0 +1,21 @@
+-- -*- mode: sql; product: postgres -*-
+
+-- Create the statistics view
+-- This is what the statistics API endpoint must expose, formated as JSON.
+CREATE VIEW statistics AS
+  SELECT name, users, maxusers, data
+    FROM hosts
+    JOIN (
+            SELECT COUNT(*)
```

Comment:
Good question.
I would expect the query planer to optimize it anyhow, because of the uniqueness of the primary key, but `COUNT(1)` isn't any harder to write.